lib/rules/no-useless-constructor.js

```diff
@@ -177,7 +179,15 @@ module.exports = {
             if (superClass ? isRedundantSuperCall(body, ctorParams) : (body.length === 0)) {
                 context.report({
                     node,
-                    messageId: "noUselessConstructor"
+                    messageId: "noUselessConstructor",
+                    suggest: [
+                        {
+                            messageId: "noUselessConstructor",
```

Comment:
Maybe we could use a new message to explain what the suggestion does ("Remove this constructor.").